PerfettoCds/Pipeline/Events/PerfettoSqlEvent.cs

```diff
@@ -0,0 +1,490 @@
+﻿using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Extensibility;
+using System;
+using Perfetto.Protos;
+
+namespace PerfettoCds.Pipeline.Events
```

Comment:
I am thinking we should refactor / separate out all C# code into a separate lib for processing a Perfetto trace totally unaware of WPA SDK or code. Thus it can be used by other projects as a sort of datalayer. Then PerfettoCDS which knows about the WPA SDK can consume this

Reply:
Done. Moved all Perfetto processing to PerfettoProcessor lib